### ISSUE TITLE
Make Name Game accessible

### ISSUE THREAD

**Issue by NikitaSalikov** [Symbiota/community-board#898]
The Name Game should have interactive and accessible elements for keyboard users and support screen-reader functionality.

location: games/namegame.php?clid=157&dynclid=0&listname=Carnegie+State+Vehicular+Recreation+area&taxonfilter=&showcommon=0
